staging/src/k8s.io/client-go/discovery/cached/memory/memcache.go

```diff
@@ -190,16 +191,29 @@ func (d *memCacheClient) refreshLocked() error {
 		return err
 	}
 
+	wg := &sync.WaitGroup{}
+	resultLock := &sync.Mutex{}
 	rl := map[string]*cacheEntry{}
 	for _, g := range gl.Groups {
 		for _, v := range g.Versions {
-			r, err := d.serverResourcesForGroupVersion(v.GroupVersion)
-			rl[v.GroupVersion] = &cacheEntry{r, err}
-			if err != nil {
-				utilruntime.HandleError(fmt.Errorf("couldn't get resource list for %v: %v", v.GroupVersion, err))
-			}
+			gv := schema.GroupVersion{Group: g.Name, Version: v.Version}
```

Comment:
nit, `v.GroupVersion` already has the string you need, right? do `gv := v.GroupVersion` instead of allocating here and stringifying in two other places?